contents/tutorials/redirect-testing.md

```diff
@@ -0,0 +1,383 @@
+
+## Creating our Next.js app and adding PostHog
+
+To start, we create our Next.js app. Run the command below, select No for TypeScript, Yes to use the `App Router`, and the default for all the other options.
+
+```bash
+npx create-next-app@latest redirect-test
+```
+
+### Setting up PostHog
+
+Next, we set up PostHog. Start by going into your new `redirect-test` folder and installing it.
+
+```bash
+cd redirect-test
+npm i posthog-js
+```
+
+Next, in the `redirect-test/app` folder, create a `providers.js` file and set up a component that returns an initialized `PostHogProvider`. You can get the project API key and instance address you need for initialization from [your project settings](https://app.posthog.com/project/settings).
+
+```js
+// app/providers.js
+'use client'
+import posthog from 'posthog-js'
+import { PostHogProvider } from 'posthog-js/react'
+
+if (typeof window !== 'undefined') {
+  posthog.init("<ph_project_api_key>", {
+    api_host: "<ph_instance_address>"
+  })
+}
+
+export default function PHProvider({ children }) {
+  return <PostHogProvider client={posthog}>{children}</PostHogProvider>
+}
+```
+
+Import the `PHProvider` component into `layout.js` and wrap your app in it.
+
+```js
+import './globals.css'
+import PHProvider from './providers'
+
+export default function RootLayout({ children }) {
+  return (
+    <html lang="en">
+      <PHProvider>
+        <body>{children}</body>
+      </PHProvider>
+    </html>
+  )
+}
+```
+
+Once set up, PostHog autocaptures usage and you can use all its tools throughout your app.
+
+## Adding test pages
+
+In the `app` folder, create two new folders named `control` and `test`. In each of them, create a basic `page.js` file with a button to capture an event. This is what the control page looks like:
+
+```js
+// app/control/page.js
+'use client'
+import { usePostHog } from "posthog-js/react";
+
+export default function Control() {
+  const posthog = usePostHog();
+
+  return (
+    <main>
+      <h1>Hello!</h1>
+      <button onClick={() => posthog.capture("main_button_clicked")}>
+        Click me!
+      </button>
+    </main>
+  );
+}
+```
+
+This is what the test page looks like:
+
+```js
+// app/test/page.js
+'use client'
+import { usePostHog } from "posthog-js/react";
+
+export default function Test() {
+  const posthog = usePostHog();
+
+  return (
+    <main>
+      <h1>Hello from the bright side!</h1>
+			<p>Clicking this button will bring you happiness</p>
+      <button onClick={() => posthog.capture("main_button_clicked")}>
+        I want to be happy!
+      </button>
+    </main>
+  );
+}
+```
+
+Now we can run `npm run dev`, go to each of our pages, `http://localhost:3000/control` and `http://localhost:3000/test`, to see that they work, and click the button to capture a custom event in PostHog (we need it to create our experiment).
+
+![Events](../images/tutorials/redirect-testing/events.png)
+
+## Creating our experiment
```

Comment:
Again, we havent introduced the term experiment yet, so feels a bit strange. Perhaps there's a more elegant way to introduce it? Or we could just stick with A/B test
```suggestion
## Creating our A/B test
```